engine/src/main/java/org/terasology/math/ChunkMath.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2013 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.math;
+
+import org.terasology.math.geom.Vector3f;
+import org.terasology.world.chunks.ChunkConstants;
+
+/**
+ * Collection of math functions.
+ *
+ * @author Benjamin Glatzel <benjamin.glatzel@me.com>
+ */
+public final class ChunkMath {
+
+    private ChunkMath() {
+    }
+
+    /**
+     * Returns the chunk position of a given coordinate.
+     *
+     * @param x The X-coordinate of the block
+     * @return The X-coordinate of the chunk
+     */
+    public static int calcChunkPosX(int x, int chunkPowerX) {
```

Comment:
chunkPower is effectively the base2 exponential representing the size of a chunk, right? Would it make sense to fetch this information automatically rather than request it in input?